contracts/FlowIDTableStaking.cdc

```diff
@@ -77,6 +77,13 @@ pub contract FlowIDTableStaking {
 
     /// The minimum amount of tokens that each node type has to stake
     /// in order to be considered valid
+    /// Keys:
+    /// 0 - Delegators
+    /// 1 - Collector Nodes
+    /// 2 - Consensus Nodes
+    /// 3 - Execution Nodes
+    /// 4 - Verification Nodes
+    /// 5 - Access Nodes
     access(account) var minimumStakeRequired: {UInt8: UFix64}
```

Comment:
We didn't have enums when we first deployed this contract, and adding one now would be a little awkward because we already have one in the `FlowEpoch` contract for it, so we don't want two different versions of it or a circular dependency.